taskwiki/util.py

```diff
@@ -441,3 +482,43 @@ def is_midnight(dt):
     """
 
     return dt.hour == 0 and dt.minute == 0 and dt.second == 0
+
+
+def taskopen(args):
+    try:
+        p = pexpect.spawn(
+            "taskopen " + args,
+            timeout=3,
+            echo=False,
+        )
+
+        prompts = [
+            "Type number\(s\): ",
+            "Config file.* does not exist. Do you want to create it? (Y/n)"
+        ]
+
+        while not p.eof():
+            index = p.expect([pexpect.EOF, *prompts])
+
+            if index == 0:
+                last_lines = p.before.splitlines()
+            else:
+                prompt = p.before.splitlines()
+                prompt += [bytes(prompts[index-1].replace('.*', ''), "utf-8")]
+
+                input = get_input(b"\n".join(prompt).decode("utf-8"))
+
+                p.sendline(input)
+
+        p.close(force=True)  # make sure the process is ended
+
+    except pexpect.exceptions.ExceptionPexpect as e:
+        if e.value.find("command was not found") > -1:
```

Comment:
```suggestion
        if "command was not found" in e.value:
```

should be equivalent a bit more readable :slightly_smiling_face: